governance/README.md

```diff
@@ -1,7 +1,74 @@
+
+### Current Members
+
+Current **Contributors** are listed in [the `CONTRIBUTORS.md` file](CONTRIBUTORS.md).
+
+### Requirements
+
+To be recognized with **Contributor** status, a community member must demonstrate a significant number of nontrivial code contributions which would benefit from access to try jobs, or consistent nontrivial participation in issue reporting and triage.
+
+### Nomination
+
+A **Maintainer** requests the addition of a new **Contributor** in the private *Maintainers* channel in Zulip, listing the number of nontrival contributions by that person. A person may also request that a **Maintainer** nominates them and vouch for them. If no objections are raised within one week, the person is added to the **Contributors** team.
+
+### Removal
+
+People will be removed from this team after six months of no activity. A majority of **Maintainers** can also vote to remove a **Contributor**.
+
+## Maintainers
+
+People that can review, approve and merge contributions to the Servo project.
+
+### Current Members
+
+Current **Maintainers** are listed in [the `MAINTAINERS.md` file](MAINTAINERS.md).
+
+### Requirements
+
+To be recognized with **Maintainer** status, a community member must demonstrate expertise in at least one area of Servo (usually via multiple code contributions or technical investigations in a particular crate or subcomponent) and perform several code reviews of nontrivial PRs with an existing reviewer shadowing.
+
+### Nomination
+
+A **Maintainer** has to nominate a person listing the main highlights of their work and their number of contributions in the private *Maintainers* channel in Zulip. Two more **Maintainers** must support the proposal. If no objections are raised within one week, the person is added to the **Maintainers** team.
+
+### Removal
+
+People will be removed from this team after one year of no activity. A majority of **Maintainers** can also vote to remove a **Maintainer**.
+
+## Technical Steering Committee Members
+
+People that have voting power in the governing body.
+
+### Current Members
+
+Current **TSC Members** are listed in [the `TSC-MEMBERS.md` file](TSC-MEMBERS.md).
+
+### Requirements
+
+To be recognized as **TSC Member**, a community member must demonstrate commitment and understanding of the Servo project by having participated as a Maintainer for a significant amount of time.
+
+### Nomination
+
+A **TSC Member** nominates a person in the private *TSC* channel in Zulip explaining the reasons and main highlights of their participation in the project. A majority of the **TSC Members** must approve the addition. If the vote succeeds and no objections are raised within one week, the person is added to the **TSC Members** team.
+
+The TSC can also nominate *Invited Experts* who are not working directly on Servo in a technical way, but can provide insight and help with coordination. The nomination process is the same as for other **TSC Members**.
```

Comment:
In this initial proposal there's no difference other than the entry path.

Do we want to make a difference about the voting power? Would it be that *Invited Experts* cannot vote? If they cannot vote, not sure why they would be TSC Members anyway, but maybe there's something we can find useful.

Reply:
So from the text it's not entirely clear to me if invited experts are actual TSC members with different requirements, or they are something similar to TSC members but not actually.

If it's the former it may be better to move this to "Requirements"?